### ISSUE TITLE
When SVGs and icons are included in links ambiguous anchor text is not being flagged

### ISSUE THREAD

**Issue by amberhinds** [equalizedigital/accessibility-checker#202]
This code does not flag ambiguous anchor text, though it should:
```
<a href="https://www-staging.nasa.gov/image-detail/x-59-overview-jun-2023-v1/" target="_self" class="button-primary button-primary-md" aria-label="">
					<span class="line-height-alt-1">Learn More</span>
					<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg"><circle class="button-primary-circle" cx="16" cy="16" r="16"></circle><path d="M8 16.956h12.604l-3.844 4.106 1.252 1.338L24 16l-5.988-6.4-1.252 1.338 3.844 4.106H8v1.912z" class="color-spacesuit-white"></path></svg>
				</a>
```

Can we set this rule up to ignore SVGs or other icons?
